stable/goldpinger/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: goldpinger
-version: 2.0.2
+version: 2.0.4
```

Comment:
please follow this: https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart